Documentation/project-docs/writing-tests.md

```diff
@@ -54,13 +54,13 @@ public async Task Headers_SetAfterRequestSubmitted_ThrowsInvalidOperationExcepti
 }
 ```
 
-# Outerloop
-This one is fairly simple but often used incorrectly. When running tests which depend on outside influences like e.g. Hardware (Internet, SerialPort, ...) and you can't mitigate these dependencies, you might consider using the `[Outerloop]` attribute for your test. 
+# OuterLoop
+This one is fairly simple but often used incorrectly. When running tests which depend on outside influences like e.g. Hardware (Internet, SerialPort, ...) and you can't mitigate these dependencies, you might consider using the `[OuterLoop]` attribute for your test. 
 With this attribute, tests are executed in a dedicated CI loop and won't break the default CI loops which get created when you submit a PR.
-To run Outerloop tests locally you need to set the msbuild property "Outerloop" to true: `/p:Outerloop=true`.
-To run Outerloop tests in CI you need to mention dotnet-bot and tell him which tests you want to run. See `@dotnet-bot help` for the exact loop names.
+To run OuterLoop tests locally you need to set the msbuild property "OuterLoop" to true: `/p:OuterLoop=true`.
```

Comment:
I think changing the casing for this property is a pretty aggressive breaking change, specially for people that run outerloop tests in unix where casing matters.

Reply:
I'm not changing the casing, I'm just fixing the occurrences where the casing is off. It was always `OuterLoop`: https://github.com/dotnet/corefx/blob/master/eng/build.sh#L57

Reply:
👍